packages/grid/_modules_/grid/constants/eventsConstants.ts

```diff
@@ -1,85 +1,431 @@
+ * Called with a [[GridColumnHeaderParams]] object.
+ * @event
+ */
 export const GRID_COLUMN_HEADER_DRAG_ENTER = 'columnHeaderDragEnter';
+
+/**
+ * Fired when the dragging of a column header ends. Called with a [[GridColumnHeaderParams]] object.
+ * @event
+ */
 export const GRID_COLUMN_HEADER_DRAG_END = 'columnHeaderDragEnd';
 
+/**
+ * Fired when the selection state of one or multiple rows changes.
+ * Called with a [[GridSelectionModelChangeParams]] object.
+ * @event
+ */
 export const GRID_SELECTION_CHANGED = 'selectionChange';
 
+/**
+ * Fired when the current page has changed. Called with a [[GridPageChangeParams]] object.
+ * @event
+ */
 export const GRID_PAGE_CHANGED = 'pageChange';
+
+/**
+ * Fired when the page size has changed. Called with a [[GridPageChangeParams]] object.
+ * @event
+ */
 export const GRID_PAGESIZE_CHANGED = 'pageSizeChange';
 
-export const GRID_ROWS_SCROLL = 'scrolling:rows';
-export const GRID_ROWS_SCROLL_END = 'scroll:rowEnd';
+/**
+ * Fired during the scroll of the grid viewport. Called with a [[GridScrollParams]] object.
+ * @event
+ */
+export const GRID_ROWS_SCROLL = 'rowsScroll';
+
+/**
+ * Fired when scrolling to the bottom of the grid viewport. Called with a [[GridRowScrollEndParams]] object.
+ * @event
+ */
+export const GRID_ROWS_SCROLL_END = 'rowsScrollEnd';
 
+/**
+ * Fired when a `mousedown` DOM event happens in the column header separator.
+ * Called with a [[GridColumnHeaderParams]] object.
+ * @event
+ */
 export const GRID_COLUMN_SEPARATOR_MOUSE_DOWN = 'columnSeparatorMouseDown';
+
+/**
+ * Fired during the resizing of a column. Called with a [[GridColumnResizeParams]] object.
+ * @event
+ */
 export const GRID_COLUMN_RESIZE = 'columnResize';
+
+/**
+ * Fired when a column is resized. Called with a [[GridColumnResizeParams]] object.
+ * @event
+ */
 export const GRID_COLUMN_RESIZE_COMMITED = 'columnResizeCommited';
+
+/**
+ * Fired when the user starts resizing a column. Called with an object `{ field: string }`.
+ * @event
+ */
 export const GRID_COLUMN_RESIZE_START = 'columnResizeStart';
+
+/**
+ * Fired when the user starts resizing a column. Called with an object `{ field: string }`.
+ * @event
+ */
 export const GRID_COLUMN_RESIZE_STOP = 'columnResizeStop';
 
+/**
+ * Fired when the user ends resizing a column.
+ * @event
+ */
 export const GRID_COLUMN_ORDER_CHANGE = 'columnOrderChange';
 
+/**
+ * Fired when the rows are updated.
+ * @event
+ */
 export const GRID_ROWS_UPDATED = 'rowsUpdated';
+
+/**
+ * Fired when the rows are updated.
+ * @event
+ */
 export const GRID_ROWS_SET = 'rowsSet';
+
+/**
+ * Fired when the grid is emptied.
+ * @event
+ */
 export const GRID_ROWS_CLEARED = 'rowsCleared';
+
+/**
+ * Fired when the columns state is changed.
+ * Called with an array of strings correspoding to the field names.
+ * @event
+ */
 export const GRID_COLUMNS_UPDATED = 'columnsUpdated';
 
+/**
+ * Fired after the sort model is changed but before applying the new model.
```

Comment:
```suggestion
 * Fired when the sort model change.
```

Reply:
Fixed! I used the "changes" word instead to keep the same pattern.